fe1-web/src/core/network/ingestion/ExtendedMessage.ts

```diff
@@ -113,6 +119,22 @@ export class ExtendedMessage extends Message implements ProcessableMessage {
   }
 
   public toState(): ExtendedMessageState {
-    return JSON.parse(JSON.stringify(this));
```

Comment:
Explicit return values are most likely easier to understand and are also needed given the new `toState()` methods